apps/api/src/app/shared/dtos/step-filter.ts

```diff
@@ -104,13 +97,21 @@ class PreviousStepFilterPart extends BaseFilterPart {
   stepType: PreviousStepTypeEnum;
 }
 
+class TenantFilterPart extends BaseFieldFilterPart {
+  @ApiProperty({
+    enum: [FilterPartTypeEnum.TENANT],
+    description: 'Only on integrations right now',
```

Comment:
What do you guys think about this? because we are not having tenant on step filters yet.

Reply:
I would say, that we are actually allowing to do that from the API, and the API docs are actually for those who use it, so I don't think we need to mention that it's only for integrations, because we can set it now on the steps right? or we don't have the logic yet?

Reply:
We don't have that logic yet for steps. It is still the filters structure from before. And as far as I know its not a part of the cycle right now...